mne/io/base.py

```diff
@@ -1482,7 +1484,9 @@ def ch_names(self):
     @property
     def times(self):
         """Time points."""
-        return self._times
+        out = np.arange(self.n_times) / float(self.info['sfreq'])
+        out.flags['WRITEABLE'] = False
+        return out
```

Comment:
I am a bit worried about this. If a code accesses times in a loop it will recompute many times. This was like this years ago and we did it this way to avoid the issue. One way would be to cache the out (ie compute times the first time you access it and then keep it for future use).

what do you think @larsoner ?

Reply:
Indeed see links above I test that use case explicitly in this PR so at least our internal uses are not a problem with Epochs.

We could cache it but then we're in a bad place where we need to both avoid using it internally *and* keep track of it using `self._update_times`. So I'd rather not do it, and instead tell users if they get slowdowns from using `raw.times` themselves they should get it once. Internally we can make sure we don't use it any more than necessary.